src/exports.jl

```diff
@@ -1356,12 +1356,14 @@ export tetrahedron
 export tighten_simulation
 export to_elementary_symmetric
 export tor
+#export toric_covered_scheme
```

Comment:
```suggestion
```
I do not think it makes sense to keep this commented code.

Reply:
Agreed. Just removed it by accepting Benjamin's suggestion above.